fnl/hotpot/config.fnl

```diff
@@ -4,14 +4,14 @@
 ; These options are only applied to hotpots toolchain, not anything
 ; under api.compile, etc etc.
 (local default-options {:compiler {:modules {}
-                                   :macros {:env :_COMPILER}}})
+                                   :macros {:env :_COMPILER}}
+                        :provide_require_fennel false})
```

Comment:
Option name?

Reply:
"provides_fennel" "provides_require_fennel" "preload_fennel"?

provide*s* matches macro*s*, module*s* but not compiler*~~s~~*.